system/jlib/jlog.hpp

```diff
@@ -1540,6 +1570,32 @@ struct LogAccessConditions
         }
     }
 
+    void addSortByCondition(LogAccessMappedField knownCol, const char * fieldName, SortByDirection direction)
+    {
+        //if (!isEmptyString(fieldName))
+        {
+            SortByCondition condition;
+            condition.byKnownField = knownCol;
+            condition.fieldName = fieldName;
```

Comment:
bug: this looks problematic. It's storing this by ptr, but doesn't own the address (const char *) that's being passed in here.
I think you need to change SortByCondition.fieldName to a StringAttr or similar.